lib/std/os/linux/test.zig

```diff
@@ -165,6 +165,15 @@ test "sigset_t" {
     try expectEqual(sigset[2], 0);
 }
 
+test "sysinfo" {
+    var info: linux.Sysinfo = undefined;
+    const result: usize = linux.sysinfo(&info);
+    try expect(std.os.linux.E.init(result) == .SUCCESS);
+
+    try expect(info.mem_unit > 0);
```

Comment:
Should this be:

```suggestion
    try expect(info.mem_unit >= std.heap.page_size_min);
```

? Or are there cases where this won't hold? If the latter, it might be nice to add a comment explaining that.

Reply:
Ok, makes sense. I'm satisfied with leaving it as is then.